src/fullnode.rs

```diff
@@ -54,6 +55,7 @@ impl Fullnode {
         keypair: Keypair,
         network_entry_for_validator: Option<SocketAddr>,
         sigverify_disabled: bool,
+        json_rpc_enabled: bool,
```

Comment:
Like the RPU, I'd prefer this always be enabled. If there's some port conflict, can we resolve that elsewhere?

Reply:
Done! It wasn't really a port-conflict issue, but a thread blocking issue.